addons/point_of_sale/migrations/9.0.1.0.1/pre-migration.py

```diff
@@ -33,6 +38,30 @@
 }
 
 
+def populate_pos_order_tax_ids(cr):
+    if not openupgrade.is_module_installed(cr, "pos_pricelist"):
+        _logger.error(
```

Comment:
This is not an error condition, so please don't log as such. And even more, no there's a return, so no table is populated at all. I think you should simply `return`.

Reply:
Thanks for your review. I saw it as an error, because having pos_pricelist installed, I was loosing all my taxes. But in a regular usage, it is not the case.

PR updated.